schedule/reschedule.py

```diff
@@ -415,7 +405,7 @@ def reschedule_card(cid, fsrs: FSRS, recompute=False):
         fsrs.set_fuzz_factor(cid, card.reps)
         new_ivl = fsrs.fuzzed_next_interval(s)
 
-        if fsrs.reschedule_threshold > 0 and not fsrs.apply_easy_days:
+        if fsrs.reschedule_threshold > 0:
```

Comment:
This should be reverted. If we use the threshold, easy days can't work because easy days adjusts the card's interval within the fuzz range, which is very small.

Reply:
Done.